iceoryx_posh/test/moduletests/test_roudi_cmd_line_parser.cpp

```diff
@@ -42,7 +43,10 @@ bool operator==(const CmdLineArgs_t& lhs, const CmdLineArgs_t& rhs)
 } // namespace config
 } // namespace iox
 
-
+namespace iox
```

Comment:
Here I would prefer just using `using namespace iox::config`. Just as a suggestion for the future

Reply:
So you want to move the using namespace here which is at line 27 or to remove the namespace iox ?

Reply:
I would remove the namespace completely and just use a test namespace were you use `using namespace iox::config` so that you do not have to write this over and over again.

Reply:
Looks like it's done.